diff --git a/node_modules/react-native-tab-view/lib/module/TabView.js b/node_modules/react-native-tab-view/lib/module/TabView.js
index 079ee72..0f97590 100644
--- a/node_modules/react-native-tab-view/lib/module/TabView.js
+++ b/node_modules/react-native-tab-view/lib/module/TabView.js
@@ -30,6 +30,7 @@ export function TabView({
   tabBarPosition = 'top',
   animationEnabled = true,
   overScrollMode,
+  offscreenPageLimit,
   options: sceneOptions,
   commonOptions
 }) {
@@ -79,6 +80,7 @@ export function TabView({
       onTabSelect: onTabSelect,
       animationEnabled: animationEnabled,
       overScrollMode: overScrollMode,
+      offscreenPageLimit: offscreenPageLimit,
       style: pagerStyle,
       layoutDirection: direction,
       children: ({
diff --git a/node_modules/react-native-tab-view/src/TabView.tsx b/node_modules/react-native-tab-view/src/TabView.tsx
index 07c6570..c8964a5 100644
--- a/node_modules/react-native-tab-view/src/TabView.tsx
+++ b/node_modules/react-native-tab-view/src/TabView.tsx
@@ -68,6 +68,7 @@ export function TabView<T extends Route>({
   tabBarPosition = 'top',
   animationEnabled = true,
   overScrollMode,
+  offscreenPageLimit,
   options: sceneOptions,
   commonOptions,
 }: Props<T>) {
@@ -129,6 +130,7 @@ export function TabView<T extends Route>({
         onTabSelect={onTabSelect}
         animationEnabled={animationEnabled}
         overScrollMode={overScrollMode}
+        offscreenPageLimit={offscreenPageLimit}
         style={pagerStyle}
         layoutDirection={direction}
       >
